src/main/java/com/ibm/watson/developer_cloud/personality_insights/v2/model/ContentItem.java

```diff
@@ -25,77 +25,20 @@
 public class ContentItem extends GenericModel {
   private String charset;
   private String content;
-  private String contenttype;
+  private String contentType;
   
   @JsonAdapter(LongToDateTypeAdapter.class)
   private Date created;
-  private Boolean forward;
+  private boolean forward;
```

Comment:
Please use the boxed `Boolean` instead of the primitive `boolean` for model classes. The reason for this is the difference in default values: If a value (e.g. `forward`) is not present in the JSON response, a `Boolean` will become `null`, while a `boolean` will become `false`.